CIScripts/Test/PesterLogger/RemoteLogCollector.Tests.ps1

```diff
@@ -146,19 +173,45 @@ Describe "RemoteLogCollector" {
     }
 
     BeforeAll {
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "Sess1", Justification="Pester blocks are handled incorrectly by analyzer.")]
         $Sess1 = $null
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "Sess2", Justification="Pester blocks are handled incorrectly by analyzer.")]
         $Sess2 = $null
-        $IntegrationTest=$false # TODO: Will change when selfcheck PR is merged.
-        if ($IntegrationTest) {
-            $Sess1 = New-PSSession -ComputerName localhost
-            $Sess2 = New-PSSession -ComputerName localhost
-        }
+    }
+}
+
+Describe "RemoteLogCollector - with actual Testbeds" -Tags CI, Systest {
+
+    BeforeAll {
+        $Sessions = New-RemoteSessions -VMs (Read-TestbedsConfig -Path $TestenvConfFile)
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "Sess1", Justification="Pester blocks are handled incorrectly by analyzer.")]
+        $Sess1 = $Sessions[0]
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "Sess2", Justification="Pester blocks are handled incorrectly by analyzer.")]
+        $Sess2 = $Sessions[1]
     }
 
     AfterAll {
-        if ($IntegrationTest) {
-            Remove-PSSession $Sess1
-            Remove-PSSession $Sess2
+        if (-not (Get-Variable Sessions -ErrorAction SilentlyContinue)) { return }
+        Remove-PSSession $Sessions
+    }
+
+    BeforeEach {
+        $DummyLog1 = ((Get-Item $TestDrive).FullName) + "\remotelog.txt"
+        "remote log text" | Out-File $DummyLog1
+        $DummyLog2 = ((Get-Item $TestDrive).FullName) + "\remotelog_second.txt"
+        "another file content" | Out-File $DummyLog2
+    }
+
+    AfterEach {
+        Remove-Item "TestDrive:/*" 
+        if (Get-Item function:Write-Log -ErrorAction SilentlyContinue) {
+            Remove-Item function:Write-Log
         }
     }
+
+    Test-MultipleSourcesAndSessions -DescribeName "RemoteLogCollector - with actual Testbeds"
```

Comment:
We wonder if there could be a way to get DescribeName. Is it possible to use `Get-CurrentPesterScope` here?